apps/public-reference/pages/listing.tsx

```diff
@@ -256,6 +256,7 @@ export default class extends Component<ListingProps> {
                       <UnitTables
                         units={listing.units}
                         unitSummaries={listing.unitsSummarized.byUnitType}
+                        disableAccordion={listing.disableUnitsAccordion}
```

Comment:
nitpick: should this be Accordion = disabled or something similar? I try to avoid having things phrased negatively in general, and it seems like there might eventually be more than two states?

Reply:
Well if we go with HTML conventions, everything's enabled by default unless there's a specific `disabled` attribute. I think this boolean makes sense, but the term "accordion" could be renamed perhaps. Maybe `disableExpansion` ?